src/main/java/calculator/OperatorType.java

```diff
@@ -0,0 +1,30 @@
+package calculator;
+
+import java.util.Arrays;
+import java.util.function.BiFunction;
+
+enum OperatorType {
+    PLUS((operand1, operand2) -> operand1 + operand2, "+"),
+    MINUS((operand1, operand2) -> operand1 - operand2, "-"),
+    MULTIPLY((operand1, operand2) -> operand1 * operand2, "*"),
+    DIVIDE((operand1, operand2) -> operand1 / operand2, "/");
+
+    private BiFunction<Double, Double, Double> expression;
+    private String operator;
+
+    OperatorType(BiFunction<Double, Double, Double> expression, String operator) {
+        this.expression = expression;
+        this.operator = operator;
+    }
+
+    public static OperatorType validateOperator(String input) {
```

Comment:
```suggestion
    public static OperatorType of(String operator) {
```

`validateOperator` 라는 네이밍은 검증의 의미를 포함하는 것으로 보일 수 있어요.
팩토리 메서드의 역할을 하고 있다는 것을 이름으로 나타내 보는 것이 어떨까요?